addons/project_issue/project_issue_config_settings_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+    	<record id="view_config_settings_form_inherit_project_issue" model="ir.ui.view">
+            <field name="name">project issue settings</field>
+            <field name="model">project.config.settings</field>
+            <field name="inherit_id" ref="project.view_config_settings"/>
+            <field name="arch" type="xml">
+                <xpath expr="//group[@name='timesheets']" position="attributes">
```

Comment:
Well I don't really understand the meaning of all this. 

group[@name='project_management']" position="after" should do the thing